lib/transactional.pm

```diff
@@ -172,8 +179,9 @@ sub trup_call {
                 die 'Abort dialog not shown' unless wait_serial('Abort');
                 send_key 'ret';
             }
-        }
-        else {
+        } elsif ($cmd =~ /^-n/) {
```

Comment:
This puts some restrictions on the caller as it is now required to use non-interactive as first argument

Reply:
From the `transactional-update --help` output:
```bash
# transactional-update --help
...
Syntax: transactional-update [option...] [general-command...] [package-command]
        transactional-update [option...] standalone-command
…
Options:
--interactive, -i          Use interactive mode for package command
--non-interactive, -n      Use non-interactive mode for package command
```

My new regular expression:
```perl
$cmd =~ /-\w*n\w* pkg/
```

Samples I was testing on:
```bash
-n pkg install q # Valid
-q -n pkg install q # Valid
-qn pkg install q # Valid
-nq pkg install q # Valid
reboot -nq pkg install q # Valid
pkg -n install -q # Not valid
-q pkg -n install q # Not valid
```